com.unity.netcode.gameobjects/Tests/Runtime/OwnerPermissionTests.cs

```diff
@@ -0,0 +1,221 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.TestTools;
+using Unity.Netcode.TestHelpers.Runtime;
+
+namespace Unity.Netcode.RuntimeTests
+{
+    public class OwnerPermissionTestComponent : NetworkBehaviour
```

Comment:
What does this component do for the test?

Reply:
Useless copy/paste from another test. Will remove. Thanks.